swagger.yaml

```diff
@@ -5415,6 +5425,19 @@ definitions:
       console_url:
         type: string
 
+  host-label-params:
+    type: object
+    required:
+      - 'label_name'
+      - 'label_set'
+    properties:
+      label_name:
+        description: Name of the label.
+        type: string
+      label_set:
```

Comment:
I think the API and implementation would be simpler if we just overwrite the list of labels instead of setting/unsetting certain labels

Reply:
You don't seem to be using this parameter in the code anyway?

Reply:
I had thought of this but label_set would be useful when we want to unset a label from a host. If we implement it to just take a list of labels, we wouldn't be able to achieve the said functionality.

Reply:
If you want to unset, just provide the list of labels without that one.  This is a short-lived process of configuring before installation - I don't expect too many updates and would rather keep the implementation and API simple.